ikas-international-end-client-is-tier1-investment-bank-python-developer-with-flask-javascript-and-sql.html

```diff
@@ -0,0 +1,37 @@
+---
+title: Python Developer with Flask - Javascript and SQL
+company: Ikas International (End client is Tier1 Investment Bank)
```

Comment:
This site requires that you disclose the name of the client. It's actually one of the few rules we have here.